applications/serial_lte_modem/src/gnss/slm_at_gnss.c

```diff
@@ -553,21 +601,8 @@ static void fix_rep_wk(struct k_work *work)
 		LOG_WRN("Failed to read GNSS NMEA data, error %d", err);
 	} else {
 		/* Report to nRF Cloud by best-effort */
-		if (nrf_cloud_ready && location_signify) {
-			err = date_time_now(&ts_ms);
-			if (err) {
-				err = nrf_cloud_rest_send_location(&rest_ctx, device_id,
-								nmea.nmea_str, -1);
-			} else if (last_ts_ms == 0 ||
-				ts_ms > last_ts_ms + REST_LOCATION_REPORT_MS) {
-				last_ts_ms = ts_ms;
-				err = nrf_cloud_rest_send_location(&rest_ctx, device_id,
-								nmea.nmea_str, ts_ms);
-			}
-			if (err) {
-				LOG_WRN("Failed to send location, error %d", err);
-			}
-		}
+		send_location(&nmea);
+
```

Comment:
Recommend to restore the old way, align with sending-control of NMEA URC next

if (nrf_cloud_ready && location_signify) {
   send_location(&nmea);
}